controllers/backup/controller.go

```diff
@@ -112,18 +124,21 @@ func (r *RepoManagerBackupReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, err
 	}
 
-	r.updateStatus(ctx, pulpBackup, metav1.ConditionFalse, "BackupComplete", "Running "+pulpBackup.Spec.DeploymentType+" dir backup ...", "BackupDir")
+	r.updateStatus(ctx, pulpBackup, metav1.ConditionFalse, "BackupComplete", "Running "+deploymentType+" dir backup ...", "BackupDir")
 	err = r.backupPulpDir(ctx, pulpBackup, backupDir, pod)
 	if err != nil {
-		r.updateStatus(ctx, pulpBackup, metav1.ConditionFalse, "BackupComplete", "Failed to backup "+pulpBackup.Spec.DeploymentType+" dir!", "FailedBackupDir")
+		r.updateStatus(ctx, pulpBackup, metav1.ConditionFalse, "BackupComplete", "Failed to backup "+deploymentType+" dir!", "FailedBackupDir")
 		return ctrl.Result{}, err
 	}
 
 	log.Info("Cleaning up backup resources ...")
 	r.updateStatus(ctx, pulpBackup, metav1.ConditionFalse, "BackupComplete", "Cleaning up backup resources ...", "DeletingBkpPod")
 	r.cleanup(ctx, pulpBackup)
+	if err := r.setStatusFields(ctx, pulpBackup, formattedCurrentTime); err != nil {
+		log.Error(err, "Failed to update pulpbackup status!")
```

Comment:
please avoid to use `pulp` on user facing messages

Reply:
my bad!